config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -24,23 +24,23 @@ base="$(dirname $0)"
 # irrelevant because the prow.sh script will pick a suitable KinD
 # image or build from source.
 k8s_versions="
-1.19
 1.20
 1.21
+1.22
 "
 
 # All the deployment versions we're testing.
 deployment_versions="
-1.19
 1.20
 1.21
+1.22
 "
 
 # The experimental version for which jobs are optional.
-experimental_k8s_version="1.21"
+experimental_k8s_version="1.22"
 
 # The latest stable Kubernetes version for testing alpha jobs
-latest_stable_k8s_version="1.20" # TODO: bump to 1.21 after testing a pull job
+latest_stable_k8s_version="1.21" # TODO: bump to 1.21 after testing a pull job
```

Comment:
You can remove the "TODO" comment.

Reply:
Or rather, turn that into "TODO: bump to 1.22 after testing a pull job". We should have done that for the "on-kubernetes-1.21" already much earlier. They were still marked as "optional" when they should have been "required".

That's a missing step 9 in https://github.com/kubernetes-csi/csi-release-tools/blob/master/SIDECAR_RELEASE_PROCESS.md#adding-support-for-a-new-kubernetes-release